### ISSUE TITLE
How to use this project to obtain different forms of given arabic word - Everything is in arabic

### ISSUE THREAD

**Issue by MonsterMMORPG** [linuxscout/sarf#2]
Everything is in Arabic and i am clueless

I have opened the project in eclipse

Disabled this important because it was giving error

![desc](https://cloud.githubusercontent.com/assets/4149474/24249007/be9345ce-0fe2-11e7-9888-092437c7cfd2.png)

Then when i run the application i get this screen

I type سيارة in to the panel and hit the button (which i assume means car but i dont know what button means)

but it gives error

![errrsad](https://cloud.githubusercontent.com/assets/4149474/24249076/ecbebfb4-0fe2-11e7-8f91-54ed3b14768a.png)

Are there any way i can use command line to print results into text files

Lets say i give word سيارة which means car

It generates all possible forms in a structure way and writes into a text file 

So like

`make_suffix.java سيارة -output myoutputfile.txt`

I need to obtain forms from the code not from the UI. because i have hundreds of thousands of words to check

**Comment by MonsterMMORPG** [comment]
Ok i have checked the application thoroughly and i concluded that it is imposible to use for non - arabic user

So i really need your help to obtain what i need

I give a word and obtain whether it is verb, adjective, adverb or noun

Then if it is noun i obtain all plural forms ,  like male, female, forms etc

If it is adjective i obtain comparative and superlative forms

Obtain its related form if exists

Obtain its all dual forms

Basically all of the forms of a given word that software can generate

I see it has many buttons and when i click it shows more info if i provide root words

I tried google translate but it is not sufficient to solve how to use the software

![trial](https://cloud.githubusercontent.com/assets/4149474/24250278/5e4e11e4-0fe7-11e7-94a4-65a57a56fe25.png)

**Comment by MonsterMMORPG** [comment]
I wonder would it possible for you write a  java application that will does this?

It can work via command line

It accepts a word, an output file name

Then it can write to that ouput file as an XML file

Like the ones you provide here : https://sourceforge.net/projects/arramooz/files/0.3/arramooz.xml.0.3.tar.bz2/download

Very well constructed, easy to parse and understand

So for each provided word, it generates all forms of that given word in such constructed matter and writes to the output file we provided

the application already has save the table feature. so it should be easier to code
